tests/dummy/app/templates/docs/buttons.md

```diff
@@ -0,0 +1,40 @@
+# Buttons
+
+`ember-caluma` provides buttons for mutating the state of work items.
+
+## Usage
+
+To use these component call them with the required parameters.
+
+### WorkItemButton
+
+The text of the button is per default the mutation name, it can be overwritten with either the `label` parameter or using the component in block form.
+
+```hbs
+<WorkItemButton @mutation="complete" @workItemId={{this.workItemId}}/>
```

Comment:
Don't use a variable in a documentation. No one knows what `{{this.workItemId}}` is supposed to be. Is it a UUID, is it a graphql ID? Just put it directly into the template..